pyFV3/__init__.py

```diff
@@ -1,6 +1,8 @@
 from ._config import DynamicalCoreConfig
 from .dycore_state import DycoreState
-from .stencils import DryConvectiveAdjustment, DynamicalCore
+from .stencils.fv_dynamics import DynamicalCore
+from .stencils.fv_subgridz import DryConvectiveAdjustment
+from .wrappers.geos_wrapper import GeosDycoreWrapper
```

Comment:
It was a, confused, attempt to prevent all of the `stencils` module from being added to the cache, thus preventing any cyclical imports. It has been changed back now.